DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests.cs

```diff
@@ -240,5 +240,38 @@ public void UpdateDelegateUserCentrePrompts_updates_user()
                 }
             }
         }
+
+        [Test]
+        public void ApproveDelegateUsers_approves_delegate_users()
+        {
+            using (var transaction = new TransactionScope())

```

Comment:
Formatting wise, you should be able to do:
```
using var transaction = new TransactionScope(); 
try ... 
```

Reply:
I prefer the explicit bracketing to make it clear where the using block is, do we prefer the alternative?

Reply:
I think the bracketing can be useful in a place where we need to distinguish or emphasise where it's used, whereas in these tests it's only causing more noise that makes it harder to follow. I'm not feeling super strongly about it so if you think for you it makes harder to follow the other way around, I'm happy to leave it (as long as there are no IDE warnings)